cfme/tests/services/test_service_catalogs.py

```diff
@@ -140,7 +140,7 @@ def test_edit_catalog_after_deleting_provider(provider, setup_provider, catalog_
     provider.delete(cancel=False)
     catalog_item.update({'description': 'my edited description'})
     flash.assert_success_message('Service Catalog Item "{}" was saved'.format(
```

Comment:
i do wonder if perhaps flash messages should support format string parameters to ensure more favorable alignment is in line with automated source code formatting